SwiftReflector/CustomSwiftCompiler.cs

```diff
@@ -87,6 +91,25 @@ public void Compile (SwiftCompilerOptions compilerOptions, bool outputIsFramewor
 			}
 		}
 
+		// TJ - let's run the files seperately and create a list of all errors
+		// coming from the Swift Compiler for easier debugging
+		(string, string) ViewAllErrors (SwiftCompilerOptions compilerOptions, bool outputIsFramework, string [] files)
+		{
+			StringBuilder sb = new StringBuilder ();
+			string pathToWrapper = null;
+			foreach (var f in files) {
+				string [] file = new string [] { f };
```

Comment:
This is part just a debugging thing that I will most likely remove in shipment, but the existing `BuildCompileArgs` method takes in a string array of files. 
Here I am just running them once at a time since if I run all of them and the first one causes an error, I will only see the error from the first file and none of the other ones.